### PR TITLE
[Text Fragments article] Add ::target-text

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add ::target-text

### CODE REVIEW

File: src/site/content/en/blog/text-fragments/index.md [hunk 1]
```diff
@@ -379,6 +379,38 @@ Feature detection is mainly intended for cases where links are dynamically gener
 (for example by search engines) to avoid serving text fragments links
 to browsers that do not support them.
 
+### Styling text fragments
+
+By default, browsers style text fragments the same way they style
+[`mark`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/mark)
+(typically black on yellow, the CSS
+[system colors](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#system_colors)
+for `mark`).
```
Comment (jpmedley) [inline src/site/content/en/blog/text-fragments/index.md:388]:
```suggestion
for `mark`.
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/text-fragments/index.md (hunk 1 above)
Comment (tomayac) [reply src/site/content/en/blog/text-fragments/index.md:388]: This leaves the opening’(‘. Do you want to remove both?

**Comment by kaycebasques** [comment]
Hello! This is an automated review. It updates automatically when code or GitHub comments in this pull request are created or updated.

## Requested changes

If there are any common problems with the content files you created or modified, they will be listed here.

### `src/site/content/en/blog/text-fragments/index.md`

* Please break lines at 100 characters maximum. Long lines can be difficult to work with in GitHub's pull request UI. The following tools can help speed up the process: [Rewrap](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap) (VS Code), [Docs To X](https://gsuite.google.com/u/0/marketplace/app/docs_to_x/243920501110) (Google Docs; only available for Googlers). Affected lines: 39, 59, 148, 150, 163, 173, 186, 192, 228, 247, 318, 332, 521, 526

**Comment by tomayac** [comment]
@kaycebasques @jpmedley This should be good to go. Thanks!

**Review by jpmedley — APPROVED** [review]

**Pull request merged**